### ISSUE TITLE
Config not required ?

### ISSUE THREAD

**Issue by seuros** [mperham/sidekiq#2811]
I recently updated sidekiq/sidekiq-pro and it seem that 'sidekiq/pro/config' is not required anymore.

```
NameError:
       uninitialized constant Sidekiq::Pro::Config
```

I had to require it in the initializer to fix this error.

**Comment by mperham** [comment]
Can you give me the backtrace so I can see what needs it?

**Comment by seuros** [comment]
sidekiq/pro/api.rb need it. line 95 and 105

**Comment by mperham** [comment]
Got it, yeah, I was trying to avoid requiring it unless necessary but forgot the client does use it.

**Comment by mperham** [comment]
Fixed in 3.0.4
